### ISSUE TITLE
ONG sending transaction has wrong metadata info

### ISSUE THREAD

**Issue by ArtemGoryunov** [trustwallet/blockatlas#898]
## Summary of Bug

ONG send transaction is marked `Claim Rewards` transaction. 

## Expected Behaviour

Every send transaction by user should be mark as a Send transaction. Check info below.

## Steps to Reproduce

____

##### Request details:
```
https://host/v1/ontology/AUyL4TZ1zFEcSKDJrjFnD7vsq5iFZMZqT7?token=ong
```

##### Response details:
```
{
  "total": 7,
  "docs": [
    {
      "id": "d3be042ae21a313bc70fbfea62d11fb32731126101859a66f523e081c3fd429c",
      "coin": 1024,
      "from": "AUyL4TZ1zFEcSKDJrjFnD7vsq5iFZMZqT7",
      "to": "Abm1FqnU4qur9bviXmrD5YnNixXGvMsi9R",
      "fee": "10000000",
      "date": 1582189609,
      "block": 7836941,
      "status": "completed",
      "type": "any_action",
      "direction": "outgoing",
      "memo": "",
      "metadata": {
        "coin": 1024,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "ong",
        "name": "Ontology Gas",
        "symbol": "ONG",
        "decimals": 9,
        "value": "400000000"
      }
    }
  ],
  "status": true
}
```

where `metadata.title` is `Claim Rewards` but it should be `Send` according to the explorer data

##### Request details:
```
https://explorer.ont.io/address/AUyL4TZ1zFEcSKDJrjFnD7vsq5iFZMZqT7/ALL/20/1
```

##### Response details:
 
<img width="1230" alt="Screenshot 2020-02-20 at 12 07 47" src="https://user-images.githubusercontent.com/725247/74919199-4f346b00-53db-11ea-9def-306de022676a.png">

**Comment by EnoRage** [comment]
https://github.com/trustwallet/blockatlas/blob/7c842154b493f5af9029e55b1db3e9a2ff3e1948/platform/ontology/model.go#L128

**Comment by Pantani** [comment]
@ArtemGoryunov can you remove sensitive information, please? like hosts

**Comment by kolya182** [comment]
One way you can get claim reward transaction is to call rpc, instead of explorer.
http://dappnode2.ont.io:20334/api/v1/smartcode/event/txhash/63d39733db24d05a4072f8edf8afe9e5a42d62849324a8f1a8f87f49e1d285df

Smart contrat 36bb5c053b6b839c8f6b923fe852f91239b9fccc  is claim contract and 0200000000000000000000000000000000000000   is  ong contract.

**Comment by EnoRage** [comment]
Closed by @Pantani #899
